### PR TITLE
SAK-28972 - Remove sakai.kernel.version from master and wherever

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: master/pom.xml [hunk 1]
```diff
@@ -75,7 +75,6 @@
   <properties>
     <sakai.version>11-SNAPSHOT</sakai.version>
     <!-- kernel only versions -->
-    <sakai.kernel.version>11-SNAPSHOT</sakai.kernel.version>
```
Comment (buckett) [inline master/pom.xml]: To make contrib tools work without having to change could you just do:
```
<sakai.kernel.version>${sakai.version}</sakai.kernel.version>
```
so they are always in sync.

**Comment by ottenhoff** [comment]
Nice can we remove this from portal skins also?  As it's now just redundant text in the footer.

**Comment by jonespm** [comment]
@ottenhoff  Yeah, I was looking at that but wondering if it would still be useful information if you had a newer kernel installed than the other tools. Also since that was referenced in code (in entitybroker and portal) it would take a more time to do a build build and test. So I wasn't personally convinced that this didn't have any value anymore to put the time into cleaning that up.

@buckett Yeah, I guess we could still retain this property there and just remove it from core. I think this came up because I forgot to change this value one release (all of these properties need to be edited manually during the release process) and it messed up someone trying to build it.

File: master/pom.xml [hunk 2]
```diff
@@ -75,7 +75,6 @@
   <properties>
     <sakai.version>11-SNAPSHOT</sakai.version>
```
Comment (buckett) [inline master/pom.xml]: Off Topic: Could this just switch to ${project.version} so it just inherits the value from the artifact version?

**Comment by buckett** [comment]
@jonespm I'm happy for sakai.kernel.version to disappear completely, was just a thought.

**Comment by jonespm** [comment]
@buckett It might be possible now, I feel like there was some edge cases that still exist because of http://jira.codehaus.org/browse/MNG-624 that make setting these versions tricky. 

There's a few properties I have to set every release.
These three: We could pull the SakaiRSF project into Sakai and that would be fixed. RSF Util we could set to a fixed version, but that would make the newer version (if any) a little harder to test. (Like reflectutils)
    <rsfutil.version>0.8.1-SNAPSHOT</rsfutil.version>
    <sakairsf.components.version>11-SNAPSHOT</sakairsf.components.version>
    <sakairsf.version>11-SNAPSHOT</sakairsf.version>

This we should just be able to delete or maybe set to project.version, I forget why it wasn't, probably just nobody put the time to do it.
    <sakai.msgcntr.version>11-SNAPSHOT</sakai.msgcntr.version>

**Comment by jonespm** [comment]
Okay, I think I've got all of the applicable changes made (finally) cleaning up the version.kernel in kernel and portal as well.

**Comment by buckett** [comment]
LGTM

**Comment by ottenhoff** [comment]
LGTM.  Ready to merge @jonespm ?

**Pull request merged**